packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -489,6 +490,9 @@ class ExpansionTile extends StatefulWidget {
   /// which means that the expansion arrow icon will appear on the tile's trailing edge.
   final ListTileControlAffinity? controlAffinity;
 
+  /// {@macro flutter.material.inkwell.hoverColor}
+  final Color? hoverColor;
```

Comment:
Should this be a MaterialStateProperty?

Reply:
I suggest to use MaterialStateProperty on background color for handle all MaterialState colors...
and remove hover color

Reply:
![Screenshot 2023-12-12 at 11 25 44](https://github.com/flutter/flutter/assets/69054810/56f95039-1cfc-420f-bdd9-5a5431aaf566)


![Screenshot 2023-12-12 at 11 23 29](https://github.com/flutter/flutter/assets/69054810/3f62fea6-e8e7-43d6-8ecb-7aea4f5d793f)

I think that he means ?

Reply:
Yep. I think so too! :)

Reply:
@M97Chahboun would you like to continue working on this?

Reply:
Yes, but I have some difficulties on tests...